dev-libs/serdisplib/files/disable-static-build.patch

```diff
@@ -0,0 +1,212 @@
+--- a/src/Makefile.in	2017-02-02 13:37:48.000000000 +0100
++++ b/src/Makefile.in	2018-05-14 20:52:34.061230980 +0200
+@@ -65,7 +65,6 @@
+ 
+ LIB_NAME=serdisp
+ 
+-LIB_STATIC=lib$(LIB_NAME).a
+ LIB_SHARED=lib$(LIB_NAME).so.$(VERSION)
+ 
+ LIB_DIR=../lib
+@@ -78,20 +77,11 @@
+ 
+ ifeq "@needs_libdl@" "true"
+  EXTRA_LIBS+=-ldl
+- ifeq "@use_sharedlib@" "false"
+-   EXTRA_LIBS_STATIC=@EXTRA_LIBS_STATIC@
+- endif
+ endif
+ 
+ 
+-# link shared or static lib to executables
+-ifeq "@use_sharedlib@" "true"
+-  LIB_SERDISP=-L$(LIB_DIR) -l$(LIB_NAME)
+-else
+-  LIB_SERDISP=$(LIB_DIR)/$(LIB_STATIC)
+-  # static execs also need to link to libraries in LDFLAGS
+-  EXTRA_LIBS_STATIC+=$(LDFLAGS)
+-endif
++# link shared lib to executables
++LIB_SERDISP=-L$(LIB_DIR) -l$(LIB_NAME)
+ 
+ 
+ ifeq "@WITH_DRIVER_SED153X@" "true"
+@@ -167,9 +157,7 @@
+ 
+ all: libraries programs
+ 
+-libraries: library.static library.shared
+-
+-library.static: $(LIB_DIR)/$(LIB_STATIC)
++libraries: library.shared
+ 
+ library.shared: $(LIB_DIR)/$(LIB_SHARED)
+ 
+@@ -178,18 +166,14 @@
+ .c.o:
+ 	$(CC) $(CFLAGS) $(DEFINES) -c $<
+ 
+-$(LIB_DIR)/$(LIB_STATIC): $(LIB_OBJECTS)
+-	$(top_srcdir)/mkinstalldirs $(top_srcdir)/lib
+-	$(AR) $(LIB_DIR)/$(LIB_STATIC) $(LIB_OBJECTS)
+-
+ $(LIB_DIR)/$(LIB_SHARED): $(LIB_OBJECTS)
+ 	$(top_srcdir)/mkinstalldirs $(top_srcdir)/lib
+ 	$(CC) -fPIC -shared $(SONAME_FLAG) -o $(LIB_DIR)/$(LIB_SHARED) $(LIB_OBJECTS) $(LDFLAGS) $(EXTRA_LIBS)
+ 	cd $(LIB_DIR) && $(LN_S) -f $(LIB_SHARED) lib$(LIB_NAME).so.$(VERSION_MAJOR)
+ 	cd $(LIB_DIR) && $(LN_S) -f lib$(LIB_NAME).so.$(VERSION_MAJOR) lib$(LIB_NAME).so
+ 
+-testserdisp: $(LIB_DIR)/$(LIB_STATIC) $(OBJECTS)
+-	$(CC) -o testserdisp $(OBJECTS) $(LIB_SERDISP) $(EXTRA_LIBS_STATIC)
++testserdisp: $(LIB_DIR)/$(LIB_SHARED) $(OBJECTS)
++	$(CC) -o testserdisp $(OBJECTS) $(LIB_SERDISP) $(EXTRA_LIBS)
```

Comment:
This change seems unnecessary as well, EXTRA_LIBS should not need to be linked directly, I think.

Reply:
Some tools do this, I just set it for the others so. I can undo this, if you think, this is a better idea?

Reply:
All tools expect one don't need it. Changed the patch.